docs/using/tutorial/moving-volume-tutorial.rst

```diff
@@ -0,0 +1,184 @@
+Before you begin, you'll need to download our sample ``YAML`` files:
+
+.. container:: hidden
+
+   .. Create the files to be downloaded with ``cURL``, but don't show download links for them.
+
+   :download:`docker-compose.yml`
+   :download:`deployment-node1.yml`
+   :download:`deployment-node2.yml`
+		 
+.. version-code-block:: console
+
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/docker-compose.yml
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node1.yml
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node2.yml
+
+.. XXX The version-code-block presents a different style to prompts, so this document would benefit from FLOC 2141 (version-prompt)
+
+The :file:`docker-compose.yml` file describes your distributed application:
+
+.. literalinclude:: docker-compose.yml
+   :language: yaml
+
+The :file:`deployment-node1.yml` file describes which containers to deploy, and where:
+
+.. literalinclude:: deployment-node1.yml
+   :language: yaml
+
+#. To deploy both your web application and server onto one of the virtual machines you have just created, use the Flocker CLI:
+
+   .. prompt:: bash you@laptop:~$
+
+      flocker-deploy 172.16.255.250 deployment-node1.yml docker-compose.yml
+
+#. Visit http://172.16.255.250/.
+   You will see the visit count displayed.
+#. Visit http://172.16.255.251/.
+   You will see that the count persists because Flocker routes the traffic from either node named in the deployment file to the one that has the application.
+
+Migrating a Container to the Second Host
+========================================
+
+The diagram below illustrates your current server-side Flocker setup:
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :alt: In the server-side Flocker setup there are two servers, one of which has two Docker containers running; one container is a running a web application, the other has a Redis database with a volume.
+   :align: center
+
+To move the container with the Redis server along with its data volume, use the :file:`deployment-node2.yml` file:
+
+.. literalinclude:: deployment-node2.yml
+   :language: yaml
+
+#. To migrate one of the containers to the second host, we'll again use the Flocker CLI:
+
+   .. prompt:: bash you@laptop:~$
+
+      flocker-deploy 172.16.255.250 deployment-node2.yml docker-compose.yml
+
+   The container on the Redis server and its volume have now both been moved to the second host, and Flocker has maintained its link to the web application on the first host:
+#. Visit http://172.16.255.250/.
+   You will see the visit count is still persisted.
+#. Visit http://172.16.255.251/.
+   You will see that the count still persists, even though the container with the volume has moved between hosts, which would not have been possible without using Flocker.
```

Comment:
This works and by SSH'ing in to my nodes I was able to check that the container and volume had actually moved. Now this is more of a marketing question, but my concern here is that these instructions don't really prove that any migration has taken place. They show your Flask app and Redis server are still running and that the data is still available - but that's exactly what would happen if the second `flocker-deploy` had silently failed to do anything.